File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let j;
    for(let i = 0; i < nums.length; i++) {
        j = i + 1
        while(j < nums.length) {
            let ans = nums[i] + nums[j];
            if (ans === target) return [i, j];
            j++;
        }
    }
};

File: 0001-two-sum/0001-two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        int pointerA = 0;
        int pointerB = nums.Length-1;

        while (pointerA < nums.Length) {
            while(pointerA != pointerB) {
            if (nums[pointerA] + nums[pointerB] == target) return new int[]{pointerA, pointerB};
            pointerB--;
            }
            pointerA++;
            pointerB = nums.Length-1;
        }
        return null;        
    }
}